de-lib/src/main/java/org/iplantc/de/admin/desktop/client/metadata/presenter/callbacks/SaveTemplateSuccess.java

```diff
@@ -0,0 +1,10 @@
+package org.iplantc.de.admin.desktop.client.metadata.presenter.callbacks;
+
+import com.google.web.bindery.autobean.shared.Splittable;
+
+import jsinterop.annotations.JsFunction;
+
+@JsFunction
+public interface SaveTemplateSuccess {
+    void resolve(Splittable result);
```

Comment:
Nope, I am getting a warning for this, but I overlooked it; maybe because it's also mixed in with other warnings for other classes I didn't update in this PR 😊 

I'll update this and the other warnings too 👍

Reply:
👍